tests/scripts/saveImages.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${scriptdir}/../../build/common.sh"
+
+docker save -o rookamd64.tar  ${BUILD_REGISTRY}/rook-amd64:latest
```

Comment:
change this to `docker tag ${BUILD_REGISTRY}/rook-amd64:latest testing/rook-amd64:latest` and the save.

also add load support in this script.